compiler/rustc_codegen_ssa/src/back/write.rs

```diff
@@ -1851,7 +1851,7 @@ fn spawn_work<'a, B: ExtraBackendMethods>(
     .expect("failed to spawn work thread");
 }
 
-enum SharedEmitterMessage {
+pub enum SharedEmitterMessage {
```

Comment:
Question: I'm probably missing something, but every other visibility change is *restricting* visibility, while this particular instance is increasing visibility. Was this intentional?

Reply:
Good catch. Not sure how that happened, but I did it with `struct Diagnostic` as well. I will revert those changes.